### PR TITLE
add craft ai sklearn model

### PR DESCRIPTION
**Pull request by Totoketchup**
closes https://github.com/craft-ai/craft-ai-client-python/issues/133

### CODE REVIEW

File: craftai/sklearn_model.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+    # Get the decision tree for all the sent data (corresponding
+    # to the DT generated at the last timestamp)
+    tree = self.craft_client.get_decision_tree(self.agent_name, last_timestamp)
+
+    # Compute the decision of this DT on the input data
+    predictions_df = self.craft_client.decide_from_contexts_df(tree, X)
+
+    # Return the predictions
+    return predictions_df["{}_predicted_value".format(self.output_name)].fillna("unknown")
```
Comment (sperretcraft) [inline craftai/sklearn_model.py]: Why do you fill NaN values ?

File: craftai/sklearn_model.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+  @staticmethod
+  def add_timestamp(df, start=0):
+    df = df.assign(ts=pd.date_range(start, periods=df.shape[0], freq="s", tz="UTC"))
+    df.set_index("ts", inplace=True)
+    return df
```
Comment (sperretcraft) [inline craftai/sklearn_model.py]: Why not just
```
df.index = pd.date_range(start, periods=df.shape[0], freq="s", tz="UTC")
```
?

File: requirements.txt [hunk 3]
```diff
@@ -10,3 +10,5 @@ pandas==0.20
 semver==2.7.7
 python-dateutil==2.7.2
 IPython==5.8.0
+scikit-learn==0.20.3
+scipy==1.2.1
```
Comment (sperretcraft) [inline requirements.txt]: why is `scipy` added to the requirements ?

File: craftai/sklearn_model.py (hunk 2 above)
Comment (Totoketchup) [reply craftai/sklearn_model.py]: Yep I'm gonna try this

File: craftai/sklearn_model.py (hunk 1 above)
Comment (Totoketchup) [reply craftai/sklearn_model.py]: Because it is the way we originally do in the pandas client

**Pull request merged**